docs/source/troubleshooting.md

```diff
@@ -1,22 +1,22 @@
 # Troubleshooting
 
-When troubleshooting, you may see unexpected behaviors or receive an error
+While troubleshooting, you may see unexpected behaviors or receive an error
 message. This section provides links for identifying the cause of the
 problem and how to resolve it.
 
 ## Behavior
 
 ### JupyterHub proxy fails to start
 
-If you have tried to start the JupyterHub proxy and it fails to start:
+If you have tried to start the JupyterHub proxy and it fails to run:
 
 - check if the JupyterHub IP configuration setting is
   `c.JupyterHub.ip = '*'`; if it is, try `c.JupyterHub.ip = ''`
 - Try starting with `jupyterhub --ip=0.0.0.0`
 
 **Note**: If this occurs on Ubuntu/Debian, check that you are using a
-recent version of [Node](https://nodejs.org). Some versions of Ubuntu/Debian come with a version
-of Node that is very old, and it is necessary to update Node.
+recent version of [Node](https://nodejs.org). Some versions of Ubuntu/Debian come with an older version
```

Comment:
This dramatically weakens the language - we want to communicate that it is very old. At the very least, it should say that it's "old" not "older"